python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_condition.py

```diff
@@ -266,6 +268,44 @@ def all_deps_match(condition: "AutomationCondition") -> "AllDepsCondition":
 
         return AllDepsCondition(operand=condition)
 
+    @public
+    @experimental
+    @staticmethod
+    def any_checks_match(
+        condition: "AutomationCondition[AssetCheckKey]", blocking_only: bool = False
+    ) -> "AnyChecksCondition":
+        """Returns an AutomationCondition that is true for an asset partition if at least one of its
+        checks evaluate to True for the given condition.
+
+        Args:
+            condition (AutomationCondition): The AutomationCondition that will be evaluated against
+                this asset's checks.
+            blocking_only (bool): Determines if this condition will only be evaluated against blocking
+                checks. Defaults to False.
+        """
+        from dagster._core.definitions.declarative_automation.operators import AnyChecksCondition
+
+        return AnyChecksCondition(operand=condition, blocking_only=blocking_only)
```

Comment:
It would be help to see an example usage of this top-level static factory method.